pybossa/forms/forms.py

```diff
@@ -93,6 +94,17 @@ class ProjectForm(Form):
                                         min_len=PROJECT_PWD_MIN_LEN,
                                         special=False)])
 
+    product = SelectField(lazy_gettext('Product'), choices=[("", "")], default="")
+    subproduct = SelectField(lazy_gettext('Subproduct'), choices=[("", "")], default="")
+    kpi = IntegerField(lazy_gettext('KPI'), widget=html5.NumberInput(), default=0)
+
+
+    def set_project_choices(self):
```

Comment:
You can define an `__init__` in the class so you don't have to call this every time you create a new form